src/charts/row.js

```diff
@@ -155,8 +155,11 @@ define(function(require) {
             },
 
             _labelsFormatPct = ({pctChange}) => {
+                const prepend = pctChange > 0 ? '+': '';
                 if(isNaN(pctChange))
                     return '----';
+                return prepend + d3Format.format(labelsNumberFormat)(pctChange) + '%';
```

Comment:
this should be moved down, but ill fix it.

Reply:
thank you